### DISCUSSION TITLE
add a option too hear the voice options  feature request

### DISCUSSION THREAD

**Discussion by tola5** [DrewThomasson/ebook2audiobook#161] [Ideas]
will be nice with a option too hear the voice one pick for the book before run the whole book

**Comment by ROBERT-MCDOWELL** [comment]
why not. once we will reach a stable release we will add it...

**Comment by ROBERT-MCDOWELL** [comment]
ok all ideas and new features for now into discussions, then the most essential ones will be back to issues as feature once we lll finish to fix the bugs and make it stable....

**Comment by ROBERT-MCDOWELL** [comment]
btw for now you can hear the voices in voices folder. it's a wav file

**Comment by ROBERT-MCDOWELL** [comment]
feature added to the next release.

**Comment by ROBERT-MCDOWELL** [comment]
available on last git and v25.2.18
